modules/geolocation/unittests.py

```diff
@@ -202,6 +202,100 @@ def test_point_set_2(self):
         actual = self.locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+        
+        
+class TestOutputAnalysis(unittest.TestCase):
```

Comment:
I think we should standardize on pytest for testing

Reply:
For the test cases I was following the format from the others that used unit test

Reply:
Out of scope of PR.

Am I hearing the dreaded word: *refactor*?